ITEM.tsv

```diff
@@ -6812,7 +6812,7 @@ ITEM_20150918_006811	A bag containing a variety of alchemical tools.
 ITEM_20150918_006812	A piece of paper swallowed by a monster. It looks like someone's portrait.
 ITEM_20150918_006813	A picture frame swallowed by a monster. There is evidence that something was put in here before.
 ITEM_20150918_006814	Pas-de-warp order to give you the future master. Sanctuary is available only when you experience problems do not mean excessive hoarding.
-ITEM_20150918_006815	A strong string obtained from monster. It is useful to tie something.
+ITEM_20150918_006815	A strong rope obtained from monster. It is useful to tie something.
```

Comment:
I think it should be "a monster" too.
Also, the second sentence could be "It is useful for tying things." or "It is useful for tying."